docs/frontend/browser-support.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Browser Support
+---
+
+# Browser Supprot
+
+This application supports the following browsers
+
+- Chrome >= 70
```

Comment:
If these are the official browsers we support, we should probably update the browser list to reflect that, https://github.com/thepracticaldev/dev.to/blob/master/.babelrc#L3